packages/flutter_localizations/lib/src/widgets_localizations.dart

```diff
@@ -45,6 +45,18 @@ class GlobalWidgetsLocalizations implements WidgetsLocalizations {
   TextDirection get textDirection => _textDirection;
   late TextDirection _textDirection;
 
+  @override
+  final String scrollUp = 'Scroll Up';
```

Comment:
@shihaohong It looks like this may be the first time we need translation in widget level, May I have some advices on how to set up the localization pipeline for this?